modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -446,17 +438,11 @@ void createExoplanetSystem(const std::string& starName) {
             "Parent = '" + starIdentifier + "',"
             "Enabled = true,"
             "Renderable = {"
-                "Type = 'RenderableOrbitDisc',"
-                "Texture = openspace.absPath("
-                    "openspace.createPixelImage('exo_habitable_zone', {0, 0.92, 0.81})"
-                "),"
-                "Size = " + std::to_string(center) + ","
-                "Eccentricity = 0,"
-                "Offset = { " +
-                    std::to_string(relativeOffset) + ", " +
-                    std::to_string(relativeOffset) +
-                "}," //min / max extend
-                "Opacity = 0.05"
+                "Type = 'RenderableHabitableZone',"
+                "Texture = openspace.absPath('" + HabitableZoneTextureFile + "'),"
+                "Luminosity = " + std::to_string(system.starData.luminosity) + ","
+                "EffectiveTemperature = " + std::to_string(system.starData.teff) + ","
+                "Opacity = 0.07"
```

Comment:
seems like all our zones have an Opacity of < 0.1 I wonder if this is something that can be put into the logic of the class instead of the user always setting such small values

Reply:
I think it will be tricky since the opacity is a property of the Renderable class.  It has it's attractions to have that property be guaranteed to be the same everywhere imho